vtysh/bgp_vty.c

```diff
@@ -4748,18 +4792,21 @@ DEFUN(no_neighbor_weight,
       NEIGHBOR_ADDR_STR2
       "Set default weight for routes from this neighbor\n")
 {
-    report_unimplemented_command(vty, argc, argv);
-    return CMD_SUCCESS;
+    return cli_bgp_neighbor_weight_execute (NULL, (char *)argv[0], NULL, false);
 }
 
-ALIAS(no_neighbor_weight,
+DEFUN(no_neighbor_weight_val,
```

Comment:
`no_neighbor_weight_val` has different parameters compared to `no_neighbor_weight` when calling `cli_bgp_neighbor_weight_execute`